core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -357,7 +357,7 @@ func (c *coordinator) ReportBlocks(
 		blocksRequested[uf] = struct{}{}
 	}
 
-	c.lggr.Trace(fmt.Sprintf("filtered eligible randomness requests: %+v", unfulfilled))
+	c.lggr.Debug(fmt.Sprintf("filtered eligible randomness requests: %+v", unfulfilled))
```

Comment:
Can we add `requestIDs` as a field to this log? Or maybe this is better, if we can search for `RequestId: <blah>`?

Reply:
Yes let me use this op to add those in as well.